src/ImageSharp/Advanced/AotCompilerTools.cs

```diff
@@ -15,6 +18,20 @@ namespace SixLabors.ImageSharp.Advanced
     /// </summary>
     public static class AotCompilerTools
     {
+        /// <summary>
+        /// Seeds the <see cref="System.Runtime.CompilerServices.Unsafe"/> calls.
+        /// </summary>
+        public static void SeedUnsafe()
```

Comment:
Looks like an implementation detail from the outside.

Can't we hide this from the outside invoking it internally in a lazy way when `Seed<TPixel>` (and other variants) are invoked?

Reply:
I'll lazy call this (once) from `Seed<TPixel>` and push.

Reply:
Calling it from the static constructor instead, since that's guaranteed to only execute once.